html-block-and-inline/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title> HTML Block and Inline </title>
+  </head>
+  <body>
+    <h1>Blue Americana Snuggie&reg; Blanket </h1>
+    <h2>by SnuggieeStore</h2>
+    <p>Price: <s>$19.99</s> $14.99</p>
+    <p>Part of our new Americana Collection, this <em>beautiful</em> blue is already a customer favorite!</p>
+    <p>Enjoy complete <strong>Freedom of Movement!</strong>
+      <p>Does not slip and slide like a regular blanket and keeps you warm from head to toe.</p>
```

Comment:
You should make this `p` element a sibling to the `p` element parent, not a child of it.